### DISCUSSION TITLE
Video Only

### DISCUSSION THREAD

**Discussion by DTIVKITT** [nilaoda/N_m3u8DL-CLI#770] [General]
Hi. Is there any way I can download only the VIDEO of a stream?
There is the option "--enableAudioOnly", but I couldn't find a "--enableVideoOnly".
Did I miss something ? Thanks

**Comment by nilaoda** [comment]
Go to https://github.com/nilaoda/N_m3u8DL-CLI

**Comment by DTIVKITT** [comment]
Thanks, but I couldn't find my answer there. :(
I'd like to download Video Only, No Audio, No subs. Only the Video file. Is that possible ?

**Reply by nilaoda** [reply]
```
N_m3u8DL-RE -sv best "your link"
```

(Only separated stream support, like MPD.)

**Comment by DTIVKITT** [comment]
Great. I was looking into the CLI page, not the RE, that's why I did not find it. I'll give it a try. Thanks for your time :)
